include/small/ibuf.h

```diff
@@ -144,6 +144,10 @@ ibuf_alloc(struct ibuf *ibuf, size_t size)
 	return ptr;
 }
 
+/** Shrink the buffer to the minimal capacity to return the memory. */
```

Comment:
Nit: The comment sounds weird. Maybe, "Shrink the buffer to the minimal possible capacity needed to store the data written to the buffer and not yet consumed."

Reply:
Done